index.html

```diff
@@ -101,7 +109,7 @@
   fi
 }
 
-VERSION="${VERSION:-latest}"
+VERSION=$(sanitized_version "${VERSION:-latest}")
```

Comment:
I guess we just have to be sure elsewhere that we don't pass something like "v1.0", or I think it'll end up as "vv1.0".

Reply:
No, `sanitized_version` takes care of that, it won’t add a `v` in front of a `v`.

Reply:
Ah I missed those two chars a that the end of the exclusion, that makes sense then thanks! :+1: